build/Dockerfile

```diff
@@ -355,6 +355,9 @@ ARG NAP_MODULES=none
 # copy oidc files on plus build
 RUN --mount=type=bind,target=/tmp [ -n "${BUILD_OS##*plus*}" ] && exit 0; mkdir -p /etc/nginx/oidc/ && cp -a /tmp/internal/configs/oidc/* /etc/nginx/oidc/
 
+# create state_files directory on plus build
+RUN --mount=type=bind,target=/tmp [ -n "${BUILD_OS##*plus*}" ] && exit 0; mkdir -p /etc/nginx/state_files/
```

Comment:
I think this will work without`--mount=type=bind,target=/tmp`, but it might be better to include it in an existing `RUN` to avoid extra layers, e.g. if you add it after line 383 it could be:

```Dockerfile
    && rm -f /etc/nginx/conf.d/* \
    if [ -n "${BUILD_OS##*plus*}" ]; then mkdir -p /etc/nginx/state_files/; fi
```

Reply:
That makes sense thanks. I've change it to this. Needed an `&&` on the last line also.